src/modules/CourseDetail/index.tsx

```diff
@@ -37,10 +38,10 @@ export function CourseDetailPage(props: { data: GetCourseResponse }) {
   const SEOConfig: NextSeoProps = {
     ...defaultSEO,
     title: course.abbrName,
-    description: course.courseDesc ?? defaultSEO.description,
+    description: course.courseDescTh + '\n' + course.courseDescEn ?? defaultSEO.description,
```

Comment:
This won't work. If both are `undefined`, the description would be `'undefined
undefined'`.

Reply:
what about
```
`${course.courseDescTh}\n${courseDescEn}`
```

Reply:
or should i stick to `course.courseDesc.Th ?? course.courseDescEn ?? defaultSEO.description`?

Reply:
> or should i stick to `course.courseDesc.Th ?? course.courseDescEn ?? defaultSEO.description`?

Personally I like this but idk if @saenyakorn would be ok